strawberryfields/decompositions.py

```diff
@@ -79,36 +107,40 @@ def takagi(N, tol=1e-13, rounding=13):
     return rl, U
 
 
-def graph_embed(mat, max_mean_photon=1.0, make_traceless=True, tol=1e-6):
-    r""" Given an symmetric adjacency matrix (in general, with arbitrary complex off-diagonal and
-    real diagonal entries),
-    it returns the squeezing parameters and interferometer necessary for
+def graph_embed(A, max_mean_photon=1.0, make_traceless=True, tol=1e-6):
+    r"""Embed a graph into a Gaussian state.
+
+    Given a graph in terms of a symmetric adjacency matrix
+    (in general with arbitrary complex off-diagonal and real diagonal entries),
+    returns the squeezing parameters and interferometer necessary for
     creating the Gaussian state whose off-diagonal parts are proportional to that matrix.
 
+    Uses :func:`takagi`.
+
     Args:
-        mat (array): square symmetric complex (or real or integer) array representing a (weighted) adjacency matrix of a graph
-        max_mean_photon (float): threshold value. It guarantees that the mode with
+        A (array[complex]): square, symmetric (weighted) adjacency matrix of the graph
+        max_mean_photon (float): Threshold value. It guarantees that the mode with
             the largest squeezing has ``max_mean_photon`` as the mean photon number
-            i.e., :math:`sinh(r_{max})^2 == max_mean_photon`
-        make_traceless (boolean): removes the trace of the input matrix, by performing the transformation
+            i.e., :math:`sinh(r_{max})^2 ==` max_mean_photon.
+        make_traceless (bool): Removes the trace of the input matrix, by performing the transformation
             :math:`\tilde{A} = A-\mathrm{tr}(A) \I/n`. This may reduce the amount of squeezing needed to encode
             the graph.
-        tol (float): the tolerance used when checking if the input matrix is symmetric: :math:`|mat-mat^T| < tol`
+        tol (float): Tolerance used when checking if the input matrix is symmetric: :math:`|A-A^T| <` tol.
```

Comment:
Following Nathan's rule, this is a sentence fragment, so doesn't need capitalization and periods (I think I have this right)
```suggestion
        tol (float): tolerance used when checking if the input matrix is symmetric: :math:`|A-A^T| <` tol
```

Reply:
thumb